recipes/ratarmountcore/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "ratarmountcore" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ratarmountcore-{{ version }}.tar.gz
+  sha256: 320c87813aae4137881ab26641efb65d6b618ced002e2550787c1c2c2f851a53
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Looks like this is pure Python and could be noarch:
```suggestion
build:
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
  noarch: python
```